tests/common/dualtor/dualtor_utils.py

```diff
@@ -31,25 +31,56 @@ def mux_server_url(request, tbinfo):
     port = utilities.get_group_visible_vars(inv_files, server, 'mux_simulator_port')
     return "http://{}:{}/mux/{}".format(ip, port, vmset_name)
 
-def _url(server_url, physical_port, action):
+def _url(server_url, physical_port=None, action=None):
     """
     Helper function to build an url for given port and target
 
     Args:
         server_url: a str, the url for mux server, like http://10.0.0.64:8080/mux/vms17-8
         physical_port: physical port on switch, an integer starting from 1
-        action: a str, either "output" or "drop"
+                        If physical_port is none, the returned url contains no '/port/action' (For polling/toggling all ports)
+        action: a str, output/drop or None. If action is None, the returned url contains no '/action'
     Returns:
         The url for posting flow update request, like http://10.0.0.64:8080/mux/vms17-8/1/drop(output)
```

Comment:
I think the convention is to use "|" , for example "this|that", to indicate either "this" or "that". The example would be better like this:
`http://10.0.0.64:8080/mux/vms17-8/1/drop|output`